api-gateway/src/application/interfaces/remove-subscription.application.interface.ts

```diff
@@ -0,0 +1,3 @@
+export interface RemoveSubscriptionApplication {
+  execute(email: string): Promise<unknown>;
```

Comment:
Just caught my eye) Why would the return type be unknown?)

Reply:
It's an API gateway, and I am not sure that it should know the correct return type in the Subscription Microservice. It adds the possibility to safely change the return type if needed without changing any code in the API gateway.